src/components/fellesStyling.css

```diff
@@ -32,6 +32,10 @@
     padding-bottom: var(--a-spacing-2);
 }
 
+.enkeltinfo_value_tillegsinfo {
```

Comment:
Ikke så fan av å endre inline-elementet til block. Jeg ville egentlig ha brukt `<Detail>` fra DS her, men sånn ting er satt opp så blir det småhacky uansett 🤔
For å slippe denne CSSen så er det mulig å bare slenge på et linjeskift egentlig, se over 🆙 🎈🏠🎈